docs/tutorials/quesion-answering-bot.md

```diff
@@ -0,0 +1,217 @@
+# Question-Answering Chatbot for documentations 
+
+```{article-info}
+:avatar: avatars/gregor.jpg
+:avatar-link: https://jobs.jina.ai
+:avatar-outline: muted
+:author: Gregor @ Jina AI
+:date: November 3, 2021
+```
+
+This tutorial will take you through the process of creating your own question-answering chatbot. 
+This is an inherently difficult task, due to the fuzzyness of human language and the infinite number of questions one could ask. </br> 
+
+One way to solve this is by predicting answers using a neural network that was trained on pairs of quesitons and their corresponding answers. In many cases such a dataset is not available, like in the case of most software documentation. Let's say we want to build a chatbot to answer questions about the jina documentation. What if I told you that there is a way to reframe this task as a search problem and that this would alleviate the need for a large dataset of matching questions and answers? </br>
+
+How you ask? <em>Let me explain!</em>
```

Comment:
```suggestion
This tutorial will take you through the process of creating your own question-answering chatbot. 
This is an inherently difficult task, due to the fuzzyness of human language and the infinite number of questions one could ask.

One way to solve this is by predicting answers using a neural network that was trained on pairs of quesitons and their corresponding answers. In many cases such a dataset is not available, like in the case of most software documentation. Let's say we want to build a chatbot to answer questions about the jina documentation. What if I told you that there is a way to reframe this task as a search problem and that this would alleviate the need for a large dataset of matching questions and answers?

How you ask? *Let me explain!*
```